witchcraft-logging-formatting/src/test/java/com/palantir/witchcraft/java/logging/format/RequestLogFormatterTest.java

```diff
@@ -38,7 +38,7 @@ void formats_correctly() {
                 .params("param", "value")
                 .build());
 
-        assertThat(formatted).isEqualTo("[2019-12-25T01:02:03Z] \"GET /some/path/value http\" 203 40 99");
+        assertThat(formatted).isEqualTo("[2019-12-25T01:02:03Z] \"GET /some/path/value http\" 203 40 99 μs");
```

Comment:
It might be nicer to put the `μs` right next to the duration like `99μs` - otherwise I think that the other numbers might be other durations.

Reply:
Hmm, that was my initial thought when I added units to the formatted trace logs, however the SI rules recommend a space (number 15 on https://physics.nist.gov/cuu/Units/checklist.html)

Reply:
Perhaps adding units to the response size value would help?
```suggestion
        assertThat(formatted).isEqualTo("[2019-12-25T01:02:03Z] \"GET /some/path/value http\" 203 40 bytes 99 μs");
```

Reply:
I would actually like that as I can never remember which of the last two refers to what